docs/user-guide.md

```diff
@@ -19,6 +19,20 @@ Below are simple steps to use this portal:
 
 {% embed url="https://www.youtube.com/playlist?list=PLJH-POb_55z8YYS_qAk_QNBQeiQ2VrtZD" %}
 
+## Compliance Toolkit Landing Page
+
+On the CTK landing page, partners can view the introduction of the CTK portal and watch a video about CTK. Additionally, they can login or register by clicking the respective buttons. The landing page also provides links to other resources such as the GitHub repository, documentation, and partner details.
+
+![CTK Landing Page1](\_images/ctk-landing-page1.png)
+
+Also, it highlights some of CTK's powerful features, like its pre-built test cases, validations, and detailed reporting.
+
+![CTK Landing Page2](\_images/ctk-landing-page2.png)
+
+It also covers the details of the SBI, Android SBI, SDK, and ABIS specs. By clicking the download option on the landing page, partners can also get the compliance toolkit apk.
```

Comment:
The landing page provides information about the SBI, Android SBI, SDK, and ABIS specifications. Additionally, partners can download the compliance toolkit APK by selecting the download option.